packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.Props.ts

```diff
@@ -42,13 +44,28 @@ export interface IDropdownProps {
   onChanged?: (option: IDropdownOption, index?: number) => void;
 
   /**
-   * Optional custom renderer for the selected dropdown item
+   * Optional custom renderer for selected option displayed in input
    */
-  onRenderItem?: IRenderFunction<IDropdownOption>;

```

Comment:
Please do add it back and mark it as @deprecated in comments, and then make sure you can fall back to it if onRenderTitle is not provided.